packages/shell-chrome/assets/panel.html

```diff
@@ -26,238 +26,273 @@
-                :title="isLatest? 'Latest Version' : `Latest Version: ${latest}`">
-                    <template x-if="version">
-                        <svg 
-                        :class="{
-                            'text-green-400': isLatest,
-                            'text-orange-400': !isLatest,
-                        }"
-                        class="mr-1.5 h-2 w-2" fill="currentColor" viewBox="0 0 8 8">
-                            <circle cx="4" cy="4" r="3" />
-                        </svg>
-                    </template>
-                    <span x-text="detected"></span>
-                </div>
-            </div>
+        get detected(){
+            if(!this.showTools){
+                return 'Alpine.js tools loading'
+            }
 
-            <div class="flex items-center space-x-2">
-                <a href="https://github.com/alpinejs/alpine" target="_blank">
-                    <svg class="w-6 h-6" fill="currentColor" viewBox="0 0 20 20" xmlns="http://www.w3.org/2000/svg"><title>Alpine.js Docs</title><path fill-rule="evenodd" d="M4 4a2 2 0 012-2h4.586A2 2 0 0112 2.586L15.414 6A2 2 0 0116 7.414V16a2 2 0 01-2 2H6a2 2 0 01-2-2V4zm2 6a1 1 0 011-1h6a1 1 0 110 2H7a1 1 0 01-1-1zm1 3a1 1 0 100 2h6a1 1 0 100-2H7z" clip-rule="evenodd"></path></svg>
-                </a>
-                <a href="https://github.com/alpinejs/alpine" target="_blank">
-                    <svg class="fill-current w-5 h-5" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><title>GitHub</title><path d="M10 0a10 10 0 0 0-3.16 19.49c.5.1.68-.22.68-.48l-.01-1.7c-2.78.6-3.37-1.34-3.37-1.34-.46-1.16-1.11-1.47-1.11-1.47-.9-.62.07-.6.07-.6 1 .07 1.53 1.03 1.53 1.03.9 1.52 2.34 1.08 2.91.83.1-.65.35-1.09.63-1.34-2.22-.25-4.55-1.11-4.55-4.94 0-1.1.39-1.99 1.03-2.69a3.6 3.6 0 0 1 .1-2.64s.84-.27 2.75 1.02a9.58 9.58 0 0 1 5 0c1.91-1.3 2.75-1.02 2.75-1.02.55 1.37.2 2.4.1 2.64.64.7 1.03 1.6 1.03 2.69 0 3.84-2.34 4.68-4.57 4.93.36.31.68.92.68 1.85l-.01 2.75c0 .26.18.58.69.48A10 10 0 0 0 10 0"></path></svg>
-                </a>
-                <!-- <a href="#">
-                    <svg class="w-5 h-5" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"><title>Reload</title><polyline points="23 4 23 10 17 10"></polyline><path d="M20.49 15a9 9 0 1 1-2.12-9.36L23 10"></path></svg>
-                </a> -->
-            </div>
-        </div>
+            return this.version? `Alpine.js v${this.version} detected` : 'Alpine.js'
+        },
 
-        <div 
-        id="components" 
-        x-data="{
-            components : [],
+        get openComponent () {
+            return this.components.filter(component => {
+                return component.isOpened
+            })[0] || {}
+        },
+    }"
+    x-init="() => {
+        setTimeout(() => {
+            showTools = true
+        }, 3000)
+    }">
```

Comment:
I'm thinking `showTools` should just be

```js
        get showTools() {
            return this.components.length > 0;
        }
```

Reply:
Yeah it defo wants to be less than an arbitrary 3secs for sure. We probably need a minimum time with the load state showing to avoid a flash of content for sites with less components to load in though.

Watcher on the components array that triggers a 500ms timeout?

Reply:
Yeah that can work I agree that it'll be quite jarring that the loading state doesn't even have a chance to bounce 😄